GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -266,8 +264,8 @@ public class Glia {
 
 // MARK: - Private
 
-private extension Glia {
-    func startObservingInteractorEvents() {
+extension Glia {
```

Comment:
Why it's not private?

Reply:
because another file (Glia.Deprecated.swift) uses it, so we need to make it available for internal usage.